core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1714,28 +1723,31 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
 
     if (!controller.isActive) {
-      val result = createPartitionsRequest.newPartitions.asScala.map { case (topic, _) =>
-        (topic, new ApiError(Errors.NOT_CONTROLLER, null))
-      }
+      val result = createPartitionsRequest.data.topics.asScala.map { topic =>
+        (topic.name(), new ApiError(Errors.NOT_CONTROLLER, null))
```

Comment:
nit: drop parenthesis after `name`. A few below as well.

Reply:
Done. Went over file and removed a bunch of empty parenthesis.